### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by Busiwa24**
In the following pull request
-I created an HTML file 
-added styling with CSS
- edited my readme file 
- run checks with linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,71 @@
-# Hello-Microverse
\ No newline at end of file
+My first Micronaut Project
+
+> This is my first project as a Microverse student.
+
+![screenshot]()
+
```
Comment (Emmyn5600) [inline README.md]: - [ ] Kindly, fix this issue of screenshot image

File: README.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+## Built With
+
+- HTML
+- CSS
+- Using VS Code
+
+## Live Demo
+
+[Live Demo Link](#)
+
```
Comment (Emmyn5600) [inline README.md]: - [x] Your live demo link is opening your project again, kindly remove it we don't need this at this time.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (Emmyn5600) [inline .gitignore:2]: - [ ] Please add test.md file in .gitignore.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @Busiwa24 

Good job so far!:clap::clap:
You have done well in implementing the project requirements:clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:
- [ ] Please use present simple tense while you are committing instead of past tense.
      For example: `Add index.html file instead of Added index.html file`.

- [ ] Please remove `Hello-Microverse folder that is included in your root folders

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (Busiwa24) [reply README.md]: done

File: .gitignore (hunk 3 above)
Comment (Busiwa24) [reply .gitignore:2]: done

File: README.md (hunk 2 above)
Comment (Busiwa24) [reply README.md]: done

**Comment by Busiwa24** [comment]
applied the requested changes  thank you

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**